frontend/src/routes/Infrastructure/Clusters/ManagedClusters/components/StatusField.tsx

```diff
@@ -157,7 +157,7 @@ export function StatusField(props: { cluster: Cluster }) {
                 maxWidth: '448px',
                 bodyContent: (
                     <>
-                        <Trans i18nKey={`cluster:status.${props.cluster?.status}`} components={{ bold: <strong /> }} />
+                        <Trans i18nKey={`cluster:status.${props.cluster?.status}.message`} components={{ bold: <strong /> }} />
```

Comment:
@Randy424 The `.message` was removed by https://github.com/stolostron/console/pull/999

Reply:
Looks good. We refer to "cluster provision" as "cluster creation" most of the time. If that is the same in the UI, then I might look at:

The cluster creation could not start because the validation of the install-config.yaml file failed.